config/open-feature/org.yaml

```diff
@@ -26,6 +26,7 @@ admins:
   - thomaspoignant
 
 # org member settings - keep alphabetical
+# add yourself here if you're interested in being an org member!
```

Comment:
Could you please add me (@oxddr) and @kamil-nowosad?

Reply:
Done!